app/src/main/java/me/saket/dank/widgets/span/CenterAlignedImageSpan.kt

```diff
@@ -46,14 +30,25 @@ class CenterAlignedImageSpan(d: Drawable): ImageSpan(d, ALIGN_BASELINE) {
   override fun draw(canvas: Canvas, text: CharSequence?,
                     start: Int, end: Int, x: Float,
                     top: Int, y: Int, bottom: Int, paint: Paint) {
-    val b = getCachedDrawable()
+    val drawable = getCachedDrawable()
+    val drawableHeight = drawable.bounds.height()
+    val translationY = (bottom - top - drawableHeight - lineSpacingExtra) / 2f
```

Comment:
The thing is extra line spacing is not applied to the last line of text. For some reason, it's included in `bottom` value (unlike ordinary line spacing) and thus it's different for last line and all other lines. So this change just "inverts" vertical skew, making icon offset negative for single-line textviews

Reply:
Good to know.

Reply:
Sorry, I forgot to mark this as resolved, I was still thinking what to do with it at the moment of posting this :)